dev_docs/tutorials/performance/running_performance_journey_in_cloud.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: kibDevTutorialRunningPerformanceJourneyInCloud
+slug: /kibana-dev-docs/tutorial/performance/running_performance_journey_in_cloud
+title: Running Performance Journey In Cloud
+summary: Learn how to run performance journey against Cloud cluster
+date: 2023-05-04
+tags: ['kibana', 'onboarding', 'setup', 'performance', 'development', 'telemetry']
+---
+
+## Overview
+As a way to better understand user experience with Kibana in cloud, we support running performance journeys against
+Cloud deployments.
+The process takes a few steps:
+- Create a cloud deployment
+- Re-configure deployment with APM enabled and reporting metrics to the monitoring cluster
+- Create a user with `superuser` role to run tests with
+- Checkout the branch that matches your cloud deployment version
+- Run the performance journey
+
+### Re-configure deployment for Kibana and Elasticsearch
+We use [kibana-ops-e2e-perf](https://kibana-ops-e2e-perf.kb.us-central1.gcp.cloud.es.io/) cluster to monitor performance testing.
+
+If you would like to report APM metrics to a different cluster, make sure to change `secret_token` and `server_url` values.
+
+#### Change Elasticsearch configuration
+In the ESS Admin Console, find your deployment and navigate to `Security` page. Click `Add Settings` under `Elasticsearch keystore` and add new entry:
+
+```
+Setting name: tracing.apm.secret_token
+Secret: CTs9y3cvcfq13bQqsB
```

Comment:
I know the intent here is  anonymous access, but I expect this is going to be flagged by token scanners.  Can we look into referencing this and `packages/kbn-journeys/journey/journey_apm_config.ts` via a store?

Reply:
Good point, making a change

Reply:
99c2d3c19a09cb61aca70ddb01029f9b69f45f02